tests/test_utils.py

```diff
@@ -199,3 +201,28 @@ def test_missing_rdkit_module_error():
         with pytest.raises(ImportError) as msg:
             _ = matchms.utils.mol_to_fingerprint(mol_input, "test", 0)
         assert expected_msg in str(msg.value), "Expected different ImportError."
+
+
+def test_looks_like_adduct():
+    """Test if adducts are correctly identified"""
+    for adduct in ["M+", "M*+", "M+Cl", "[M+H]", "[2M+Na]+", "M+H+K", "Cat",
+                   "MS+Na", "MS+H"]:
+        assert looks_like_adduct(adduct), "Expected this to be identified as adduct"
+    for adduct in ["N+", "B*+", "++", "--", "[--]", "H+M+K"]:
+        assert not looks_like_adduct(adduct), "Expected this not to be identified as adduct"
+
+
+def test_clean_adduct_examples():
+    """Test if typical examples are correctly edited."""
+    test_adducts = [("M+", "[M]+"),
+                    ("M+CH3COO-", "[M+CH3COO]-"),
+                    ("M+CH3COO", "[M+CH3COO]-"),
+                    ("M-CH3-", "[M-CH3]-"),
+                    ("M+2H++", "[M+2H]2+"),
+                    ("[2M+Na]", "[2M+Na]+"),
+                    ("2M+Na", "[2M+Na]+"),
+                    ("M+NH3+", "[M+NH3]+"),
+                    ("M-H2O+2H2+", "[M-H2O+2H]2+")]
+
+    for adduct, expected_adduct in test_adducts:
+        assert clean_adduct(adduct) == expected_adduct, "Expected different cleaned adduct"
```

Comment:
These look like great candidates for parameterized tests. Now the first item that fails will cancel all assertions after it.

Reply:
Good point! Done.